morl_baselines/multi_policy/pgmorl/pgmorl.py

```diff
@@ -202,18 +203,28 @@ def predict_next_evaluation(self, weight_candidate: np.ndarray, policy_eval: np.
         return delta_predictions, delta_predictions + policy_eval
 
 
-def generate_weights(delta_weight: float) -> np.ndarray:
+def generate_weights(delta_weight: float, dimensions: int = 2) -> np.ndarray:
```

Comment:
Back when I implemented PGMORL we didn't have `equally_spaced_weights` https://github.com/LucasAlegre/morl-baselines/blob/main/morl_baselines/common/weights.py#L39 . I think we can use this here instead of custom code. What do you think? :)